drydock_backups/README.rst

```diff
@@ -17,6 +17,7 @@ Configuration variables
 
 - **BACKUP_IMAGE**: The image used to run the cronjob. (default: `ednxops/shipyard-utils:v1.0.0`)
 - **BACKUP_CRON_SCHEDULE**: Cron schedule to run the backup. (default: `0 2 * * *`)
+- **BACKUPS_STORAGE_SERVICE**: Storage service to use. (default: `s3`) (options: `s3`, `azure`)
```

Comment:
Maybe we need to define it with `aws s3` and `azure blob` or just with the name of the cloud provider `aws` and `azure`